lib/dream_html.mli

```diff
@@ -56,6 +56,13 @@ val respond :
   node ->
   Dream.response Dream.promise
 
+val send :
```

Comment:
Can you add a `@since` tag and bump the version in `dune-project` to `3.2.0`? Remember to also do a `dune build` so that the `dream-html.opam` file is also updated. Thanks!

Reply:
Should be in now.